schemas/2015-10-01-preview/Microsoft.DataLakeAnalytics.json

```diff
@@ -0,0 +1,123 @@
+{
+	"id": "http://schema.management.azure.com/schemas/2015-10-01-preview/Microsoft.DataLakeAnalytics.json#",
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Microsoft.DataLakeAnalytics",
+	"description": "Microsoft Data Lake Analytics Resource Types",
+	"resourceDefinitions": {
+		"accounts": {
+			"type": "object",
+			"description": "Microsoft.DataLakeAnalytics/accounts",
+			"properties": {
+				"apiVersion": {
+					"enum": ["2015-10-01-preview"]
+				},
+				"type": {
+					"enum": ["Microsoft.DataLakeAnalytics/accounts"]
+				},
+				"properties": {
+					"type": "object",
+					"properties": {
+						"defaultDataLakeStoreAccount": {
```

Comment:
since the property has length limitation, can you use an "anyOf" to allow both this definition and expression?
User might want to use something like "[concat(parameters('account'), variables('name')]".
Or, simply remove the length check here and leave it for service side check.